### PR TITLE
XMake v2.2.9

### PR DESCRIPTION
**Pull request by kassane**
New version avaliable!

### CODE REVIEW

**Comment by Kapeli** [comment]
Please update the `docset.json` as well. Also please be aware that you need to update the main archive as well (`docsets/XMake/XMake.tgz`)

**Comment by kassane** [comment]
> Please update the `docset.json` as well. Also please be aware that you need to update the main archive as well (`docsets/XMake/XMake.tgz`)

Fixed!

File: docsets/XMake/docset.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "XMake",
-    "version": "2.2.8",
+    "version": "2.2.9",
     "archive": "XMake.tgz",
```
Comment (Kapeli) [inline docsets/XMake/docset.json]: You need to update the archive at this path as well.

Comment (kassane) [reply docsets/XMake/docset.json]: Changed.

File: docsets/XMake/docset.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "aliases": ["Xmake"],
     "specific_versions": [
         { 
-            "version": "2.2.8",
-            "archive": "versions/2.2.8/XMake.tgz"
+            "version": "2.2.9",
+            "archive": "docsets/XMake/XMake.tgz"
```
Comment (Kapeli) [inline docsets/XMake/docset.json]: This doesn't work. The specific version path can't be the same as the main docset path.

The main docset path (docsets/XMake/Xmake.tgz) is still unchanged.

Comment (kassane) [reply docsets/XMake/docset.json]: And now it works?
>docsets/XMake/versions/2.2.9/Xmake.tgz

File: docsets/XMake/docset.json (hunk 1 above)
Comment (Kapeli) [inline docsets/XMake/docset.json]: You need to update the archive at this path as well. `specific_versions` are completely separate from the main docset.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**